definitions/tools/cnvkit.cwl

```diff
@@ -0,0 +1,108 @@
+#! /usr/bin/env cwl-runner
+
+cwlVersion: v1.0
+class: CommandLineTool
+baseCommand: ["/usr/bin/python", "/usr/local/bin/cnvkit.py", "batch"]
+
+requirements:
+    - class: ShellCommandRequirement
+    - class: DockerRequirement
+      dockerPull: "etal/cnvkit:0.9.5"
+
+inputs:
+    tumor_bam:
+        type: File
+        inputBinding:
+            position: 1
+    normal_bam:
+        type: File
+        inputBinding:
+            position: 2
+            prefix: "--normal"
+    bait_intervals:
+        type: File
+        inputBinding:
+            position: 3
+            prefix: "--targets"
+    reference:
+        type: File
+        inputBinding:
+            position: 4
+            prefix: "--fasta"
+    access:
+        type: File?
+        inputBinding:
+            position: 5
+            prefix: "--access"
+    method:
+        type: string?
+        default: "hybrid"
+        inputBinding:
+            position: 6
+            prefix: "--method"
+    threads:
+        type: int?
+        inputBinding:
+            position: 7
+            prefix: "-p"
```

Comment:
Should there be a corresponding resource requirement for the number of cores?  For example, here's how name sort specified this:
https://github.com/genome/analysis-workflows/blob/0a9a4ce83b49ed4e7eee5bcc09d83725136a36b0/definitions/tools/name_sort.cwl#L8

Reply:
There's no explicit flag from the program itself to set this. According to the documentation, 
`-p [PROCESSES], --processes [PROCESSES]
                        Number of subprocesses used to running each of the BAM
                        files in parallel. Without an argument, use the
                        maximum number of available CPUs. [Default: process
                        each BAM in serial]`

Reply:
If we're only running on a single tumor BAM, does this option need to be exposed at all?

If it does, I think the `coresMin` resource requirement should match the integer value here one way or another.

Reply:
True, this may not be necessary here, since cromwell/cwl will take care of launching this multiple times as needed if we have multiple files. I'll remove the option.